### ISSUE TITLE
Allow root folder to be read only

### ISSUE THREAD

**Issue by Snake883** [Radarr/Radarr#8775]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

I cannot add a root folder that is read only (I made the SMB network path read only).
I assume the root cause is because I set the SMB network path to be read only.

I also received this message in the error log:
2023-07-05 13:28:47.0|Warn|RadarrErrorPipeline|Invalid request Validation failed:  -- Path: Folder '/movies_main/' is not writable by user 'abc'

When adding the root folder, there are no problems reading/viewing the folders.

### Describe the solution you'd like

I would like to add a root folder that is read only.
A read only root folder should be permissible.

### Describe alternatives you've considered

I considered letting Radarr have write access to my main movie library, but I do not want to accept the risk of Radarr losing my main movie library.

### Anything else?

I have two root folders for my media library:

1)  The main media library is the library that I rip my own.  I don't want anything to touch these folders/files, or accidentally delete/modify or lose anything.  Therefore I set the path (via SMB network share) to read only.  I only want to import the movies, and catalog the movies into Radarr.  Radarr should never touch the actual files/folders (hence I made them read only).

2)  The second root folder is for movie torrents I download.  I let Radarr and the bittorrent client manage these folders/files.  This folder is read/write.

**Comment by bakerboy448** [comment]
No plans for read only folder support; the purpose of radarr is a movie library management tool software which by definition requires RW access

**Comment by Snake883** [comment]
I believe I have found a workaround...

1. Mark the movie as "Unmonitored".  I assume this means Radarr won't attempt to download, modify, rename anything.  
2. Then manually import the movie from the read only path.

If you have a lot of movies, this can take a while.

**Comment by Snake883** [comment]
> The purpose of radarr is a movie library management tool software which by definition requires RW access

I am using Radarr to manage and catalog my movie library.  Radarr does not need read/write access to accomplish this.

I am also separating my ripped movie collection using a custom Radarr release group (which is readonly), separate of someone else's collection (read/write).

**Comment by bakerboy448** [comment]
Radarr is not a catalog tool. By definition managing requires RW access.....again no plans to support RO access....there's really changing that. Given a read only set of movies there's nothing for radarr to do anyway and no reason whatsoever for those movies to be in radarr.   

If you want to avoid downloading those or something; then create the folders in Radarr's library, library import, and unmonitor the movies to prevent any new releases from being grabbed.

========

Unmonitoring is  almost never required and is not a normal use case and indicates a users quality profiles and custom formats or existing file naming is configured in a suboptimal way thus creating the possibility for upgrades to be grabbed as the user configured and told Radarr to do

**Comment by Snake883** [comment]
Thanks for taking the time to explain.  Radarr is a wonderful tool that lets me see/track which movies I ripped, and which movies I acquired.  I am going to try your workaround suggestion by copying empty folders into the Radarr root library.  

Your suggested option might also allow me the option of monitoring for a future bluray release while I keep my personal DVD ripped release untouchable.

Thank you.
